src/mdh_bridge.py

```diff
@@ -287,6 +242,56 @@ class MDHQuery_getMimeType:
     result = MDHMimeType()
 
 
+class MDHFile(MDHObject):
+    id: bool or str = False
+    metadata: bool or MDHMetadata or [MDHMetadatum] = False
+
+
+class MDHResultSet(MDHObject):
+    fromIndex: bool or int = False
+    toIndex: bool or int = False
+    totalFilesCount: bool or int = False
+    returnedFilesCount: bool or int = False
+    timeZone: bool or str = False
+    instanceName: bool or str = False
+    fixedReturnColumnSize: bool = False  # TODO how do we do this here?
+    graphQLQuery: bool or str = False
+    graphQLVariables: bool or str = False
+    files: bool or MDHFile or [MDHFile] = False
+    graphQLDebug: bool or str = False
+    dataTypes: bool or MDHMetadatatagDataType or [MDHMetadatatagDataType] = False
+
+
+class MDHQuery_searchMetadata(MDHQuery):
+    fileIds: bool or [int] = False
+    filterFunctions: bool or [MDHFilterFunction] = False
+    filterLogicOption: bool or LogicOption = True
+    sortFunctions: bool or [MDHSortFunction] = False
+    filterLogicalIndividual: bool or str = False
+    limit: bool or int = False
+    offset: bool or int = False
+    fileSizeAsHumanReadable: bool = False  # TODO how do we do this with bool values?
+    convertDateTimeTo: bool or str = False
+
+    @property
+    def selectedTags(self) -> bool or [str]:
+        if self._selectedTags and isinstance(self._selectedTags, bool):
+            _selectedTags = []
+            for dataType in self.result.dataTypes:
+                _selectedTags.append(dataType.name)
+
+            self._selectedTags = _selectedTags
+
+        return self._selectedTags
+
+    @selectedTags.setter
+    def selectedTags(self, selectedTags) -> None:
+        self._selectedTags = selectedTags
```

Comment:
what is the point of this? What are the selected tags? The intuition would suggest that is a list of all the tags in the metadata that was collected. However, this seems to just return a list of all the data types collected. So maybe at least rename for clarification?

Reply:
the selected tags correspond to the datatype names as they are inserted after the deserialization or by hand. i means its actually an attribute just like before but returns something instead of true or false :D